packages/client-sdk/src/cloud.ts

```diff
@@ -19,7 +19,12 @@ class Cloud {
     if (this.config?.requestClass) {
       ret = this.config?.requestClass;
     } else if (env === EnvironmentType.UNI_APP) {
-      ret = UniRequest;
+      const { uniPlatform } = uni.getSystemInfoSync();
+      if (uniPlatform == 'mp-weixin') {
+        ret = WxmpRequest;
```

Comment:
Further information is needed to explain the necessity of this modification。

Reply:
At present, the code of UNIAPP will have problems when compiling into a small program, and the SDK cannot be used normally